extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -107,7 +97,10 @@ public function testRenderInfoboxDataProvider() {
 			[
 				'input' => [ ],
 				'output' => '',
-				'description' => 'Empty data should yield no infobox markup'
+				'description' => 'Empty data should yield no infobox markup',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "description" 15 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)